contracts/ethregistrar/StablePriceOracle.sol

```diff
@@ -6,44 +6,72 @@ import "./StringUtils.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
 
 interface AggregatorInterface {
-  function latestAnswer() external view returns (int256);
+    function latestAnswer() external view returns (int256);
 }
 
-
 // StablePriceOracle sets a price in USD, based on an oracle.
 contract StablePriceOracle is Ownable, PriceOracle {
     using SafeMath for *;
     using StringUtils for *;
 
     // Rent in base price units by length. Element 0 is for 1-length names, and so on.
-    uint[] public rentPrices;
+    uint256[] public rentPrices;
```

Comment:
I think we probably want to replace this with a hardcoded if-else statement for lengths 3, 4, and 5+. That lets us make the values immutable and save another chunk of gas.